strings.go

```diff
@@ -8,7 +8,7 @@ import (
 	"strconv"
 	"strings"
 
-	util "github.com/aokoli/goutils"
+	util "github.com/Masterminds/goutils"
```

Comment:
I'm assuming this is okay?

Reply:
Yeah. We should actually make sure it's always imported from the Masterminds repo now.